src/client/Client.js

```diff
@@ -310,6 +312,33 @@ class Client extends BaseClient {
     });
   }
 
+  /**
+   * Obtains a sticker from Discord.
+   * @param {Snowflake} id The sticker's ID
+   * @returns {Promise<Sticker>}
+   * @example
+   * client.fetchSticker('id')
+   *   .then(sticker => console.log(`Obtained sticker with name: ${sticker.name}`))
+   *   .catch(console.error);
+   */
+  async fetchSticker(id) {
+    const data = await this.api.stickers(id).get();
+    return new Sticker(this, data);
+  }
+
+  /**
+   * Obtains the list of sticker packs available to Nitro subscribers from Discord.
+   * @returns {Promise<Collection<Snowflake, StickerPack>>}
+   * @example
+   * client.fetchNitroStickerPacks()
+   *   .then(packs => console.log(`Available sticker packs are: ${packs.map(pack => pack.name).join(', ')}`))
+   *   .catch(console.error);
+   */
+  async fetchNitroStickerPacks() {
```

Comment:
```suggestion
  async fetchPremiumStickerPacks() {
```
Rename this perhaps for consistency? No other properties and methods use the term Nitro, they use premium.